CRM/Twingle/Exceptions/BaseException.php

```diff
@@ -1,12 +1,14 @@
 <?php
 
+namespace CRM\Twingle\Exceptions;
```

Comment:
This will have to be `Civi\Twingle\...`. Please make sure to run `civix upgrade` for adding namespace tags to the `info.xml` for classloader prefixes.

Reply:
Okay, `civix upgrade` did not add any additional namespace tags to `info.xml`.